VolodymyrKulak/HW7/07.1 PRACTICAL TASKS--3.py

```diff
@@ -0,0 +1,20 @@
+
```

Comment:
good job
look at this implementation
def count_symbols_in_row(word):
    result = {}
    for item in word:
        if item in result:
            continue
        else:
            result.update({str(item): word.count(item)})
    return result